app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/service/CharacterStoredImpl.kt

```diff
@@ -0,0 +1,65 @@
+package com.puzzlebench.clean_marvel_kotlin.data.service
+
+import android.util.Log
+import com.puzzlebench.clean_marvel_kotlin.ID_TEXT
+import com.puzzlebench.clean_marvel_kotlin.MESSAGE_EXISTING_OBJECT
+import com.puzzlebench.clean_marvel_kotlin.REALM_TAG
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import com.puzzlebench.clean_marvel_kotlin.domain.model.RealmCharacter
+import com.puzzlebench.clean_marvel_kotlin.domain.model.RealmThumbnail
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Thumbnail
+import io.realm.Realm
+import io.realm.kotlin.createObject
+import io.realm.kotlin.where
+
+class CharacterStoredImpl {
```

Comment:
Make an interface for this. The useCase will call the interface and this class will implement it

Reply:
Done